cron/bq/transfer.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bq
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/ossf/scorecard/cron/config"
+)
+
+func StartDataTransferJob(ctx context.Context, bucketURL, filename string) error {
+	projectID, err := config.GetProjectID()
+	if err != nil {
+		return fmt.Errorf("error getting ProjectId: %w", err)
+	}
+	bq, err := bigquery.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("failed to create bigquery client: %w", err)
+	}
+
+	datasetName, err := config.GetBigQueryDataset()
+	if err != nil {
+		return fmt.Errorf("error getting BigQuery dataset: %w", err)
+	}
+	tableName, err := config.GetBigQueryTable()
```

Comment:
I think you might need a partition decorator as well: https://cloud.google.com/bigquery/docs/managing-partitioned-table-data#using_a_load_job ?

Reply:
Hmm, I thought it should pick the partition up using `date` column. I'm flag guarding the load job for now - it'll help with the failing tests and also allow me to manually test `bq load` before enabling it in prod.

Reply:
Yes, please test it. I'm not sure the load job is smart enough to do this without specifying the date partition (and the documentation doesn't seem to suggest this either).